pkgs/applications/science/astronomy/gildas/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, fetchurl, gtk2 , pkgconfig , python27 , gfortran , python27Packages , lesstif , cfitsio , getopt , perl , groff }:
+
+stdenv.mkDerivation rec {
+  version = "feb17d";
```

Comment:
Note that versions in Nix *must* start with a digit. Otherwise `nix-env` will consider it part of the package name. So e.g. `nix-env -i gildas` won't work.